### ISSUE TITLE
Create docker container for the API.

### ISSUE THREAD

**Issue by mesprague** [learn-to-code-denver/learncoderate#5]
Create a docker file and resulting image for students to run the api server in.

**Comment by ab7** [comment]
I believe this issue has been addressed. @jenyacazacu is there anything else you'd like to do to the Docker file before we close this one out?

**Comment by mesprague** [comment]
I think the docker file still needs some work. The application install path should be different and the way it creates the node user as a system user with a standard home directory isn't great.

**Comment by jenyacazacu** [comment]
what would you recommend @mesprague ?

**Comment by ab7** [comment]
How do you guys feel about opening new issues to address the points @mesprague brought up? I would say this ticket is complete based on the title and description.

**Comment by jenyacazacu** [comment]
yeah lets crete new issues, or maybe use a google docs spreadsheet to keep track of issues/user stories, that might be more appropriate

**Comment by mesprague** [comment]
Specific recommendations for the Docker file include:

Different installation point. Something like /srv/node/learncoderate/api and /srv/http/learncoderate/webclient

Setup permissions/acls to run as api and serve the html as the nobody user instead of creating a node user

Install build tools and other global packages through the os package manager where possible instead of npm.

**Comment by jenyacazacu** [comment]
right on
